src/FormBuilder/resources/js/form-builder/src/blocks/fields/phone/settings.tsx

```diff
@@ -0,0 +1,46 @@
+import {__} from '@wordpress/i18n';
+import defaultSettings from '../settings';
+import {FieldBlock} from '@givewp/form-builder/types';
+import Edit from './Edit';
+import {Icon} from '@wordpress/icons';
+import {Path, SVG} from '@wordpress/components';
+
+/**
+ * @unreleased
+ */
+const settings: FieldBlock['settings'] = {
+    ...defaultSettings,
+    title: __('Phone', 'give'),
```

Comment:
Nice catch! I agree it would be better. However, the same thing is happening with the email block. So, thinking about keeping the consistency, I believe that we need to update the email block name as well. What are your thoughts on it?

![image](https://github.com/impress-org/givewp/assets/16308864/ba1eb4d5-281c-49e7-a5cc-4715fa964c1d)

Reply:
Ah yeah, we can circle back to that one day haha!

Reply:
Resolved in 8401b98cda3228c20ed8c98152745717c78c2c57